### PR TITLE
[Merged by Bors] - Disabling default features support in bevy_ecs, bevy_reflect and bevy

### PR DESCRIPTION
**Pull request by mockersf**
# Objective

- Fix disabling features in bevy_ecs (broken by #5630)
- Add tests in CI for bevy_ecs, bevy_reflect and bevy as those crates could be use standalone

### CODE REVIEW

**Comment by mockersf** [comment]
bors try

**Comment by mockersf** [comment]
bors run was to check that config changes were correct
it doesn't add much duration:
Screenshot 2022-09-15 at 19 10 11

File: .github/workflows/validation-jobs.yml [hunk 1]
```diff
@@ -154,3 +154,23 @@ jobs:
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+      - name: Install alsa and udev
+        run: sudo apt-get update; sudo apt-get install --no-install-recommends libasound2-dev libudev-dev
+        if: runner.os == 'linux'
```
Comment (MinerSebas) [inline .github/workflows/validation-jobs.yml:171]: This step is unnecessary here.

**Review by james7132 — APPROVED** [review]

**Review by devil-ira — APPROVED** [review]

**Pull request closed without merging**